src/styles/blockButtonStyle.ts

```diff
@@ -0,0 +1,34 @@
+export type Size = 'xs' | 'sm' | 'md' | 'lg';
+export type Style = 'solid' | 'outlined';
+export type Hierarchy = 'accent' | 'primary' | 'secondary' | 'tertiary';
+
+interface BlockButtonStyleType {
+  size: Record<Size, string>;
+  variant: {
+    solid: Record<Hierarchy, string>;
+    outlined: Record<Hierarchy, string>;
+  };
+}
+
+export const blockButtonStyle: BlockButtonStyleType = {
+  size: {
+    xs: 'py-(--gap-3xs) px-(--gap-xs) rounded-(--radius-xs) label-xs',
+    sm: 'py-(--gap-2xs) px-(--gap-sm) rounded-(--radius-2xs) label-sm',
+    md: 'py-(--gap-xs) px-(--gap-lg) rounded-(--radius-2xs) label-md',
+    lg: 'py-(--gap-sm) px-(--gap-2xl) rounded-(--radius-3xs) label-xs',
+  },
```

Comment:
아하 이부분을 `rounded-(--radius-xs)` 형태로 작성하셔서 radius 적용이 안되어있던 것 같습니다 
radius는 제가 피그마 형식에 맞춰서 토큰을 만들어두었었는데 오히려 적용에 헷갈릴 수도 있었을 것 같네요..!
그냥 `radius-xs` 형태로 작성하시면 잘 적용될 겁니다!

```suggestion
export const blockButtonStyle: BlockButtonStyleType = {
  size: {
    xs: 'py-(--gap-3xs) px-(--gap-xs) radius-xs label-xs',
    sm: 'py-(--gap-2xs) px-(--gap-sm) radius-2xs label-sm',
    md: 'py-(--gap-xs) px-(--gap-lg) radius-2xs label-md',
    lg: 'py-(--gap-sm) px-(--gap-2xl) radius-3xs label-xs',
  },
```

Reply:
[e8e8baf](https://github.com/JECT-Study/JECT-Official-WebSite-Client/pull/30/commits/e8e8bafd569c6552ef50d9e888a91ae7cc573e19)에 적용했습니다! 친절한 설명 감사합니다 :)